proto/storage/cve.proto

```diff
@@ -189,6 +189,7 @@ message ImageCVEV2 {
 
   string component_id = 13; // @gotags: sql:"fk(ImageComponentV2:id),index=btree"
   string advisory = 14; // @gotags: search:"Advisory"
+  string advisory_link = 15;
```

Comment:
I didn't really want to add the search tag to fields in the Advisory message because I don't know what implications it would have, especially when search is not required. I can change that if it's rather harmless

Reply:
I find it unlikely we would search on the link anyway.

@ksurabhi91 When this is merged and you put in the graphql changes, can you also update this to be a struct for advisory that contains a name and a link?  I think that stuff is out of scope for @RTann as the only reason he needed to even do this part was because we convert back and forth between objects.

Reply:
@ksurabhi91 I am updating helper function that renders advisory link in UI to expect `"advisory": null` or:

```json
"advisory": {
    "advisoryId": "RHSA-2024:0041",
    "advisoryLink": "https://access.redhat.com/errata/RHSA-2024:0041"
}
```

Reply:
Going to resolve this, so I can merge, but I'll ping this to the channel